app/components/pan-zoom.jsx

```diff
@@ -126,16 +127,17 @@ class PanZoom extends React.Component {
     this.continuousZoom(0);
   }
 
-  zoomReset() {
+  zoomReset(props) {
+    props = props || this.props;
     this.setState({
       viewBoxDimensions: {
-        width: this.props.frameDimensions.width,
-        height: this.props.frameDimensions.height,
+        width: props.frameDimensions.width,
```

Comment:
Testing with the current PRN project (https://pan-zoom.pfe-preview.zooniverse.org/projects/vrooje/planetary-response-network-and-rescue-global-caribbean-storms-2017/classify?env=production), if you rotate or zoom in the subject image and hit the reset button, it'll throw ` TypeError: Cannot read property 'width' of undefined`. I think the issue is this line, but it's a little hard for me to be sure looking at the minified file.

Reply:
My bad. I broke the handler for that button.

Reply:
Should be fixed by f5066ed